HelmSetup.md

```diff
@@ -112,50 +112,50 @@ After deployed, visit devlake: https://devlake-0.example.com:8443, and grafana a
+| grafana.useExternal           | If use external grafana server                       | false                    |
+| grafana.externalUrl           | external grafana server if use external              | ""                       |
+| lake.storage.class            | storage class for lake's volume                      | ""                       |
+| lake.storage.size             | volume size for lake's data                          | 100Mi                    |
+| lake.image.repository         | repository for lake's image                          | apache/devlake           |
+| lake.image.pullPolicy         | pullPolicy for lake's image                          | Always                   |
+| lake.loggingDir               | log dir for the lake server                          | /app/logs                |
+| lake.loggingLevel             | log level for the lake server                        | info                     |
+| lake.dotenv                   | initial configurations for injecting to lake's .env  | see Values.yaml          |
+| ui.image.repository           | repository for ui's image                            | apache/devlake-config-ui |
+| ui.image.pullPolicy           | pullPolicy for ui's image                            | Always                   |
+| ui.basicAuth.enabled          | If the basic auth in ui is enabled                   | false                    |
+| ui.basicAuth.user             | The user name for the basic auth                     | "admin"                  |
+| ui.basicAuth.password         | The password for the basic auth                      | "admin"                  |
+| service.type                  | Service type for exposed service                     | NodePort                 |
+| service.uiPort                | Node port for config ui                              | 32001                    |
+| service.ingress.enabled       | If enable ingress                                    | false                    |
+| service.ingress.enableHttps   | If enable https                                      | false                    |
+| service.ingress.className     | Name for ingressClass. leave empty for using default | ""                       |
+| service.ingress.hostname      | The hostname/domainname for ingress                  | localhost                |
+| service.ingress.prefix        | The prefix for endpoints, currently not used         | /                        |
+| service.ingress.tlsSecretName | The secret name for tls's certificate for https      | ""                       |
+| service.ingress.httpPort      | The http port for ingress                            | 80                       |
+| service.ingress.httpsPort     | The https port for ingress                           | 443                      |
+| option.TZ                     | The timezone                                         | Asia/Shanghai            |
```

Comment:
@klesh @warren830 What's your opinion?

Reply:
UTC has a special meaning for developers.
It would be better to choose a neutral timezone to make it clear that it should be set by users to their own timezone.
I suggest that we use `America/Cayman` as the placeholder